conans/test/functional/profiles/test_python_profiles.py

```diff
@@ -0,0 +1,99 @@
+import os
+import textwrap
+import unittest
+
+from importlib.util import spec_from_file_location, module_from_spec
+
+from conans.client.tools import environment_append
+from conans.test.utils.tools import TestClient
+
+
+class Profile(dict):
+
+    def __init__(self):
+        self["settings"] = {}
+        self["options"] = {}
+        self["env"] = {}
+
+    @property
+    def settings(self):
+        return self["settings"]
+
+    def update(self, profile):
+        for k, v in profile.items():
+            if k in ["settings", "options", "env"]:
+                self[k].update(v)
+            else:
+                self[k] = v
+
+
+def _load_module_from_file(filename, parent_profile):
+    name = os.path.splitext(os.path.basename(filename))[0]
+    spec = spec_from_file_location(name, filename)
+    config = module_from_spec(spec)
+    for key, value in parent_profile.items():
+        setattr(config, key, value)
```

Comment:
here attributes coming from parent profile are set so they are available in the module before loading it